src/app/public/modules/file-attachment/file-drop.component.html

```diff
@@ -69,10 +69,12 @@
           <input
             type="text"
             class="sky-form-control"
+            [attr.aria-label]="'skyux_file_attachment_file_upload_link_input' | skyLibResources"
             [attr.placeholder]="'skyux_file_attachment_file_upload_link_placeholder' | skyLibResources"
             [(ngModel)]="linkUrl"
+            (blur)="linkBlur()"
```

Comment:
Would you mind naming this method `onLinkBlur` to follow Angular's suggested naming conventions?
https://angular.io/guide/styleguide#dont-prefix-output-properties
> Do name event handler methods with the prefix `on` followed by the event name.

Reply:
Done